projects/tesseract-ocr/build.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -eu
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+cd $SRC
+tar zxf leptonica-1.77.0.tar.gz
+cd leptonica-1.77.0
+./configure
+make -j$(nproc)
+make install
+ldconfig
+
+cd $SRC/tesseract
+./autogen.sh
+CXXFLAGS="$CXXFLAGS -D_GLIBCXX_DEBUG" ./configure --disable-graphics
+make -j$(nproc)
+
+cd $SRC/tesseract-ocr-fuzzers
+
+mv $SRC/tessdata $OUT
+export TESSDATA_PREFIX=`realpath $OUT/tessdata`
+
+$CXX $CXXFLAGS \
+    -DTESSDATA_PREFIX="\"$TESSDATA_PREFIX\"" \
+    -I $SRC/tesseract/src/api \
+    -I $SRC/tesseract/src/ccstruct \
+    -I $SRC/tesseract/src/ccmain \
+    -I $SRC/tesseract/src/ccutil \
+     $SRC/tesseract-ocr-fuzzers/fuzzer-api.cpp -o $OUT/fuzzer-api \
+     $SRC/tesseract/src/api/.libs/libtesseract.a \
+     /usr/local/lib/liblept.a \
+     /usr/lib/x86_64-linux-gnu/libtiff.a \
```

Comment:
Hardcoded lib path is not portable. But I could not find a better way to find static library paths.

Reply:
Another alternative might be to build it and install to a specific directory

Reply:
I don't think tiff, jpeg and png are actually used by the fuzzer, they are only needed to satisfy linker dependencies, so from the perspective of code coverage/fuzzer efficacy this is not required.

A portable but clumsy approach ```$(locate libtiff.a)```.

Please let me know if you want me to build the 3 libraries from scratch.